ui/components/app/transaction-icon/index.scss

```diff
@@ -0,0 +1,8 @@
+.transaction-icon {
```

Comment:
[REQ] lets use the name of the component as the filename instead of index, so that we can more easily search/and find these and understand where they're coming from in source maps

Reply:
done in the latest commit